food-truck-frontend/src/components/SubscriptionList.tsx

```diff
@@ -1,63 +1,35 @@
 import React from 'react';
-import api from "../util/api";
+import Truck from "../domain/Truck";
+import {loadSubscriptionsForUser} from "../api/UserApi";
 
 type SubscriptionProps = {
-        username: string
+  username: string
 }
 
```

Comment:
I kind of gave up on this file since I was planning on replacing it later for the user view stuff